yolox/core/trainer.py

```diff
@@ -265,6 +265,10 @@ def after_iter(self):
             )
 
             if self.rank == 0:
+                if self.args.logger == "tensorboard":
+                    self.tblogger.add_scalar("train/lr", self.meter["lr"].latest, self.epoch + 1)
+                    for k, v in loss_meter.items():
+                        self.tblogger.add_scalar(f"train/{k}", v.latest, self.epoch + 1)
```

Comment:
IMHO, It should be self.iter here, not epoch. WDYT @sauravkdeo

Reply:
yes, You are right, `self.progress_in_iter` will capture more granular details.